staging/src/k8s.io/mount-utils/mount.go

```diff
@@ -66,6 +66,10 @@ type Interface interface {
 	// care about such situations, this is a faster alternative to calling List()
 	// and scanning that output.
 	IsLikelyNotMountPoint(file string) (bool, error)
+	// CanSafelySkipMountPointCheck indicates whether this mounter returns errors on
+	// operations for targets that are not mount points. If this returns true, no such
+	// errors will be returned.
+	CanSafelySkipMountPointCheck() bool
```

Comment:
can this be a package private function? Not sure if external consumers should be able to consume it.

Reply:
I would tend to agree; this looks a lot like an implementation detail that probably isn't very useful or straightforward to use by a library consumer.

But, because this interface exposes `IsLikelyMountPoint`, I think it's important to expose this function as well. IMO that function should also probably not be exposed, but I think that's outside the scope of this PR.

Reply:
IsLikelyNotMountPoint has a specific use-case where we can detect mount points faster and avoid reading /proc/mounts. So, if you are 100% sure; that a given path is not a bind mount, IsLikelyNotMountPoint should work, it is also used in other places in the use cases.

I think we should keep it package private. Also, if there is a use-case in the future, making it public should be easier than making it private.

Reply:
My point was that `IsLikelyNotMountPoint` shouldn't be called in certain circumstances, based on the result of `CanSafelySkipMountPointChecks`, as in `CleanupMountPoints`. I don't feel strongly either way about making it private, so I'll make it private. I agree that it's easy to make it public later if need be.